Exercise_6/ReversePolishNotation.js

```diff
@@ -0,0 +1,43 @@
+function calculate(argument) {
+    const invalidArgumentError = new Error('The argument is invalid');
+
+    function isNumber(sign) {
+        const isNumberRegex = new RegExp("[0-9]");
```

Comment:
The most correct check is this one:
`!isNaN(parseFloat(sign)) && isFinite(sign)`
Because `Number(sign) == sign` will not pass the linter.